index.js

```diff
@@ -245,11 +262,13 @@ function normalizeArguments(url, opts) {
 		opts.method = opts.method || 'POST';
 
 		if (isPlainObj(body)) {
-			opts.headers['content-type'] = opts.headers['content-type'] || 'application/x-www-form-urlencoded';
+			if (opts.headers['content-type'] === undefined) {
+				opts.headers['content-type'] = 'application/x-www-form-urlencoded';
+			}
 			body = opts.body = querystring.stringify(body);
 		}
 
-		if (!opts.headers['content-length'] && !opts.headers['transfer-encoding'] && !isStream.readable(body)) {
+		if (opts.headers['content-length'] === undefined && !opts.headers['transfer-encoding'] && !isStream.readable(body)) {
```

Comment:
Nope, I put `=== undefined` because I want the user to be able to prevent a specific head (here `content-length`) to be defined.